Added UG for AnnotationLoaded Event and Annotations comment for .NET MAUI PDFViewer

KSilambharasanSF4548 opened the pull request:
Changes :
1. UG documentaton for Annotations Loaded event 
2. UG documentation for annotations comment

KSilambharasanSF4548 committed:
Added UG for liquid glass effect

KSilambharasanSF4548 committed:
Added annotation loaded comment UG

KSilambharasanSF4548 committed:
added annotations comment UG

KSilambharasanSF4548 committed:
removed unwanted changes

SyncfusionBuild commented:
The running **[CI Job](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4874/console)** is terminated due to changes committed on the source branch for this Merge Request and CI triggered for latest commit.

KSilambharasanSF4548 committed:
Update comment formatting in Annotations-Comment

SyncfusionBuild commented:
The running **[CI Job](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4875/console)** is terminated due to changes committed on the source branch for this Merge Request and CI triggered for latest commit.

KSilambharasanSF4548 committed:
updated annotations comments

KSilambharasanSF4548 committed:
Modify description for annotations comments feature

Updated the description to reflect the addition of comments or replies to annotations in the Syncfusion .NET MAUI PDF Viewer.

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4889/console)**

KSilambharasanSF4548 committed:
Removed unwanted changes

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4890/console)**

SyncfusionBuild commented:
**CI Status: FAILURE :x:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Failure Reason: [CI Failure with violation](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4890/artifact/cireports/)<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4890/)<br/>Technical Error(s): 0<br/>Spelling Error(s): 0<br/>Image Alt Text Error(s): [0](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4890/artifact/cireports/ImageValidation/AltTextValidation.html)<br/>Image Size Error(s): [0](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4890/artifact/cireports/ImageValidation/ImageSizeValidation.html)<br/>Image Name Error(s): [0](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4890/artifact/cireports/ImageValidation/ImageNameValidation.html)<br/>Front matter Error(s): [0](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4890/artifact/cireports/FrontMatterValidation/FrontMatterValidation.html)<br/>File Path Error(s): [1](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4890/artifact/cireports/FilePathValidation/FilePathValidation.html)<br/>CodeSnippet File Path Missing Error(s): [0](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4890/artifact/cireports/CodeSnippetFilePathValidation/CodeSnippetFilePathValidation.html)<br/>Toc Anchor Tags Validation Error(s): [0](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4890/artifact/cireports/AnchorTagsValidation/AnchorTagsValidation.html)<br/>MD Tag Validation Error(s): [0](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4890/artifact/cireports/MDTagsValidation/MDTagsValidation.html)<br/>**

KSilambharasanSF4548 committed:
annotations comments images

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4891/console)**

KSilambharasanSF4548 committed:
added images for annotations comments

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4892/console)**

KSilambharasanSF4548 committed:
Delete Document-Processing/PDF/PDF-Viewer/maui/Images/Annotations/desktop-comment.png

KSilambharasanSF4548 committed:
Delete Document-Processing/PDF/PDF-Viewer/maui/Images/Annotations/Mobile-Comment.jpg

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4909/console)**

KSilambharasanSF4548 committed:
added comments images for desktop and mobile

KSilambharasanSF4548 committed:
Added images for liquid glass

SyncfusionBuild commented:
The running **[CI Job](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4910/console)** is terminated due to changes committed on the source branch for this Merge Request and CI triggered for latest commit.

SyncfusionBuild commented:
**Build Status: INQUEUE :clock3:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4911/console)**

KSilambharasanSF4548 committed:
added liquid glass

KSilambharasanSF4548 committed:
Add images for Liquid glass effect on desktop and mobile

SyncfusionBuild commented:
The running **[CI Job](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4912/console)** is terminated due to changes committed on the source branch for this Merge Request and CI triggered for latest commit.

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4913/console)**

KSilambharasanSF4548 committed:
Replace static images with GIFs in annotations

Updated image formats for comment panel instructions.

KSilambharasanSF4548 committed:
Rename desktp-comment.gif to desktop-comment.gif

SyncfusionBuild commented:
The running **[CI Job](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4914/console)** is terminated due to changes committed on the source branch for this Merge Request and CI triggered for latest commit.

SyncfusionBuild commented:
**CI Status: ABORTED :x:<br/>Source Compilation: Not Started :x:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4914/)<br/>**

KSilambharasanSF4548 committed:
Delete Document-Processing/PDF/PDF-Viewer/maui/Images/Annotations/desktop-comment.gif

KSilambharasanSF4548 committed:
added image

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4918/console)**

KSilambharasanSF4548 committed:
Delete Document-Processing/PDF/PDF-Viewer/maui/Images/Annotations/desktop-comment.gif

KSilambharasanSF4548 committed:
added comment panel image

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4919/console)**

SyncfusionBuild commented:
The running **[CI Job](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4919/console)** is terminated due to changes committed on the source branch for this Merge Request and CI triggered for latest commit.

SyncfusionBuild commented:
**CI Status: ABORTED :x:<br/>Source Compilation: Not Started :x:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4919/)<br/>**

KSilambharasanSF4548 committed:
Delete Document-Processing/PDF/PDF-Viewer/maui/Liquid-Glass-Effect.md

KSilambharasanSF4548 committed:
Delete Document-Processing/PDF/PDF-Viewer/maui/Images/desktop-liquidglass.gif

KSilambharasanSF4548 committed:
Delete Document-Processing/PDF/PDF-Viewer/maui/Images/mobile-liquidglass.gif

SyncfusionBuild commented:
**CI Status: ABORTED :x:<br/>Source Compilation: Not Started :x:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4926/)<br/>**

SyncfusionBuild commented:
The running **[CI Job](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4927/console)** is terminated due to changes committed on the source branch for this Merge Request and CI triggered for latest commit.

SyncfusionBuild commented:
**Build Status: INQUEUE :clock3:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4928/console)**

SyncfusionBuild commented:
**CI Status: ABORTED :x:<br/>Source Compilation: Not Started :x:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4928/)<br/>**

KSilambharasanSF4548 committed:
Refine AnnotationsLoaded event description

Removed redundant sentence in AnnotationsLoaded event section.

KSilambharasanSF4548 committed:
Update debug message for loaded annotations

KSilambharasanSF4548 commented on Document-Processing/PDF/PDF-Viewer/maui/Annotation-Collection.md:
```diff
@@ -39,4 +39,25 @@ private void OnDocumentLoaded(object sender, EventArgs e)
+
+private void OnAnnotationsLoaded(object? sender, EventArgs e) 
+{ 
+    Debug.WriteLine("All existing annotations have been loaded."); 
```
removed bro

SyncfusionBuild commented:
**CI Status: ABORTED :x:<br/>Source Compilation: Not Started :x:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/4932/)<br/>**

manojSF3414 commented on Document-Processing/PDF/PDF-Viewer/maui/Annotations-Comment.md:
```diff
@@ -0,0 +1,93 @@
+
+
+## Comment panel
+Annotation comments can be added to the PDF using the comment panel. Comment panel can be opened by using Built-in toolbar , In the toolbar you can see the comments button in the primary toolbar for desktop and top toolbar for mobile. A comment panel will appear.
```
remove space after toolbar

KSilambharasanSF4548 committed:
Fix punctuation and capitalization in comment panel section

Corrected punctuation and capitalization in comment panel instructions.

manojSF3414 commented on Document-Processing/PDF/PDF-Viewer/maui/Annotations-Comment.md:
```diff
@@ -0,0 +1,93 @@
+
+
+## Comment panel
+Annotation comments can be added to the PDF using the comment panel. Comment panel can be opened by using Built-in toolbar, In the toolbar you can see the comments button in the primary toolbar for desktop and top toolbar for mobile. A comment panel will appear.
```
these lines are not clear it is said that can be opened by using built in toolbare and says buttons are there and it says comment panel will apprear. 
Rewrtie it clearly mention when will comment panel appear.

manojSF3414 reviewed (changes requested):
Hi @KSilambharasanSF4548  
Added comments.
Kindly address these feedbacks

KSilambharasanSF4548 committed:
Add link for Annotation Comments in TOC

KSilambharasanSF4548 committed:
Delete Document-Processing/PDF/PDF-Viewer/maui/Images/Annotations/desktop-comment.gif

KSilambharasanSF4548 committed:
Delete Document-Processing/PDF/PDF-Viewer/maui/Images/Annotations/mobile-comment.gif

KSilambharasanSF4548 committed:
Added annotation comments image

KSilambharasanSF4548 committed:
Changes added for annotations comments

KSilambharasanSF4548 committed:
Enhance comment panel description in documentation

Expanded explanation of the comment panel functionality.

Bharathi4332 committed:
Merge branch 'development' of https://github.com/syncfusion-content/document-processing-docs into Maui-Pdf

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [Maui-Pdf](https://ci.syncfusion.com/job/Content/job/document-processing-docs/5055/console)**
